requirements.txt

```diff
@@ -1,4 +1,3 @@
-Sphinx<5.0.0
-furo<2022.4.8
-myst-parser<0.18
-sphinx-lint<1
```

Comment:
It looks like we are not longer including `sphinx-lint` in the new requirements file? I think we should keep it if we plan to keep the `make check` rule.

Reply:
Oops! Yep it needs to be there. Thanks!